base_layer/core/migrations/00000000000000_diesel_initial_setup/up.sql

```diff
@@ -0,0 +1,36 @@
+-- This file was automatically created by Diesel to setup helper functions
```

Comment:
This file seems to be in the wrong directory. Perhaps you ran the diesel_cli in the wrong place?

Reply:
Haha thanks, Missed this one. Created a few of them in wrong places....